Purchases/SwiftSources/LocalReceiptParsing/LocalReceiptParser.swift

```diff
@@ -18,11 +18,11 @@ internal enum IntroEligibilityStatus: Int {
          eligible
 }
 
-@objc public class LocalReceiptParser: NSObject {
+@objc(RCLocalReceiptParser) internal class LocalReceiptParser: NSObject {
     
-    @objc public func checkTrialOrIntroductoryPriceEligibility(withData data: Data,
-                                                               productIdentifiers: [String],
-                                                               completion: ([String : Int], Error?) -> Void) {
+    @objc internal func checkTrialOrIntroductoryPriceEligibility(withData data: Data,
```

Comment:
do you need to specify it for all methods? or since the class is internal already, you don't need to indicate anything

Reply:
yeah, no need to specify it at all. I had specified it for the sake of making sure that this was working with an internal class, but it's redundant

Reply:
do you mind if I clean that up in one of the follow-up PRs?

Reply:
yeah, no worries. It was more like a question I had hehe